<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User_information</class>
 <widget class="QDialog" name="User_information">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/pic_main/25.jpg);</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>451</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="m_user">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;楷体&quot;;
</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="m_password">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;楷体&quot;;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="set_password"/>
    </item>
    <item>
     <widget class="QLabel" name="m_auth">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;楷体&quot;;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="m_money">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;楷体&quot;;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="m_sex">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;楷体&quot;;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="set_email"/>
    </item>
    <item>
     <widget class="QLabel" name="m_tel">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;楷体&quot;;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="set_tel"/>
    </item>
    <item>
     <widget class="QLabel" name="m_explaim">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;楷体&quot;;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="set_explain"/>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="confirm_Button">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>550</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确认修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="supply_Button">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>550</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>完善资料</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
